### DISCUSSION TITLE
节点上的文字，超长时省略出三个点，鼠标移动到文字上时给出完整的tip提示，这个功能在X6中有吗？有的话，怎么配置？多谢。

### DISCUSSION THREAD

**Discussion by zhaoweikai** [antvis/X6#1905] [Q&A]
节点上的文字，超长时省略出三个点，鼠标移动到文字上时给出完整的tip提示，这个功能在X6中有吗？有的话，怎么配置？多谢。

**Comment by zhaoweikai** [comment]
研究了下，可以通过自定义HTML节点的方式实现，但感觉比较麻烦。X6自身有这个功能吗？感觉这个功能很常用。

**Reply by kylewuyinhai** [reply]
可以通过textWrap这个属性来配置https://x6.antv.vision/zh/docs/api/registry/attr#textwrap
![image](https://user-images.githubusercontent.com/50390162/158964360-08be9fd6-8c61-4ba2-ac75-8ec7dea8cca3.png)
